### DISCUSSION TITLE
Hi Jan!

### DISCUSSION THREAD

**Discussion by LUISMAGAIVER341** [JanDeDobbeleer/oh-my-posh#3422] [Ideas]
Hi Jan! Can i help with the posh?
Sadly i doens't code with Go, Lua and others.
But, i know basics of python, HTML, and CSS, and, with my determination i can create a shell, specially for posh, with extra features, that runs on mac, windows, and linux!
I love posh, and i want collaborate, cuz this project can grow big too much, and much!
Plz tell Yes! Posh turns the ugly shell, into something wonderful!

**Comment by JanDeDobbeleer** [comment]
@LUISMAGAIVER341 there's always something to look at, there's no public roadmap (yet), but you can always look for issues coming in or spelling errors in the docs for example.

**Reply by LUISMAGAIVER341** [reply]
Ohh guy, i'm sorry. I can't.
